index.rst

```diff
@@ -48,7 +48,353 @@
+* A DataLink "links service" endpoint to connect image dataset IDs from ObsCore queries
+* Image-retrieval signed URLs for the actual image access, returned from the "links service"
+* An image-cutout service pointed to by metadata from the "links service"
+
+
+CAOM2 metadata
+""""""""""""""
+
+It's not a primary goal of PP Preview to exercise the creation and provision of CAOM2 metadata for
+acquired data, but it would be a useful stretch goal.
+
+Architecture-team guidance for how this metadata will be created and stored in a database will
+be required in order for progress to be made in this area.
+
+Catalog Data
+^^^^^^^^^^^^
+
+The final version of PP Preview will have to demonstrate the ability to migrate data from
+the APDB to the PPDB.
+The PPDB is expected to be a Postgres database.
+It is not completely clear whether it's the same database as will be used for the image metadata.
+
+RSP Portal Aspect
+-----------------
+
+As in DP0.2 and as as planned for ops, the Portal Aspect obtains data through a combination
+of direct access to IVOA-standard interfaces, mostly importantly TAP queries and DataLink links services,
+and through DataLink-compatible but not specifically standardized data services discovered via DataLink.
+
+Image Data
+^^^^^^^^^^
+
+The Portal for PP Preview will access image data from TAP queries to ObsCore-formatted tables.
+
+The equivalent of data-rights-user access to post-embargo data will be via a true "ObsTAP" service,
+with the table named ``ivoa.ObsCore``.
+That table will represent the image data in the post-embargo Butler repository.
+
+Early access to pre-release data will be via tables representing ``/repo/embargo`` or any other Butler
+repositories that are relevant.
+Users will be able to select these tables in the PP Preview Portal's TAP screen, and when they do so,
+as is already true for the prototype ``live ObsCore`` service, the Portal will display the same
+search screen that it does for true "ObsTAP" services.
+
+If and when CAOM2 metadata becomes available, it will be searchable via TAP in the Portal Aspect, but the
+PP Preview era Portal will not provide any dedicated CAOM2-based search capabilities.
+
+Catalog Data
+^^^^^^^^^^^^
+
+As noted above, toward the end of PP Preview the DIA catalog products will be replicated to the Postgres PPDB.
+Portal access to that data will simply be via TAP in a way that has largely been exercised in DP0.2 and DP0.3.
+
+As notec abov, what is new here is the evolving nature of the ``DiaObject`` table, in particular.
+While that's not *per se* a problem for the Portal Aspect, we know that there are both user expectations
+and actual requirements (*vide supra*) on the reproducibility of queries.
+
+(TBS: list of requirements)
+
+PP Preview will allow these use cases to be exercised in the Portal Aspect, and will help us to
+determine whether we need a dedicated UI capability to support capturing PPDB queries in a
+reproducible form.
+
```

Comment:
While I think it's clearly out of scope for the preview, do I remember correctly that we expect the Portal to provide some sort of interface to the Alert Archive?